spec/policies/division_policy_spec.rb

```diff
@@ -10,27 +10,41 @@
   context 'being a member of a division' do
     let(:user) { create(:user, :member, division: division) }
 
-    permit_actions [:index, :show]
+    it 'can not index' do
+      expect(described_class.new(user, Division).index?).to be_falsey
+    end
```

Comment:
The factored up `permit_actions` and `forbit_actions` tests are all testing against the shared `subject` which is built around an model instance.  The `index` action tests need to use a policy instance around the class.